active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,419 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Cleanlab to Determine Examples to Collect Additional Labels for in an Active Learning Loop to Improve Classification Models "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how to compute an active learning score for every datapoint in your dataset, and shows how a classification model improves after relabeling examples for multiple rounds. The examples in the dataset will have been annotated by multiple annotators, and not all examples have to be labeled at the beginning. The computed active learning score will indicate if you should collect more labels for an already-labeled example, or get a label for an example that has not been labeled.\n",
+    "\n",
+    "This notebook will demonstrate the following steps:\n",
+    "\n",
+    "1. Train a model on the current consensus labels to obtain out-of-sample predicted probabilities for each datapoint\n",
+    "2. Compute active learning scores for every datapoint, this score indicates the quality and confidence of each datapoint's label, where a low active learning score indicates that we should get more\n",
+    "3. Collect additional labels for the examples with the lowest active learning scores\n",
+    "4. Repeat the steps above with a new set of consensus labels obtained after collecting more labels\n",
+    "\n",
+    "In this example we used a modified version of the [Wall-Following Robot Navigation](https://www.openml.org/search?type=data&sort=runs&status=any&qualities.NumberOfClasses=gte_2&qualities.NumberOfInstances=between_1000_10000&id=1526) dataset, where we synthetically generated multiple annotations for each datapoint with some noise. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import dependencies and get data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.ensemble import ExtraTreesClassifier\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from cleanlab.multiannotator import get_majority_vote_label, get_label_quality_multiannotator, get_active_learning_scores\n",
+    "\n",
+    "from utils.model_training import train_model\n",
+    "from utils.active_learning import get_idx_to_relabel, setup_next_iter_data, add_new_annotator\n"
```

Comment:
`get_idx_torelabel()` seems like crucial function and should probably be defined in this notebook to be explicit.

Reply:
also should we call it "to_relabel" or "to_label".

We are collecting a new label for these datapoints, whereas "relabel" sounds to me like overwriting the existing label

Reply:
`idx_to_label` makes sense! Agree that `relabel` could be confusing.